packages/unplugin-vue-i18n/src/core/resource.ts

```diff
@@ -302,6 +277,10 @@ export function resourcePlugin(
       }
     },
 
+    loadInclude(id: string) {
```

Comment:
This was necessary to make things work in `rspack` and probably improves performance for `webpack`, it seems like the plugin was handling more files than expected (such as css files as in the repro shared in https://github.com/intlify/bundle-tools/issues/417#issuecomment-2705395571)